zetaclient/chains/bitcoin/rpc/rpc.go

```diff
@@ -4,14 +4,24 @@ import (
 	"fmt"
 
 	"github.com/btcsuite/btcd/btcjson"
+	"github.com/btcsuite/btcd/chaincfg"
 	"github.com/btcsuite/btcd/chaincfg/chainhash"
 	"github.com/btcsuite/btcd/rpcclient"
 	"github.com/pkg/errors"
 
+	"github.com/zeta-chain/zetacore/zetaclient/chains/bitcoin"
 	"github.com/zeta-chain/zetacore/zetaclient/chains/interfaces"
 	"github.com/zeta-chain/zetacore/zetaclient/config"
 )
 
+const (
+	// feeRateCountBackBlocks is the default number of blocks to look back for fee rate estimation
+	feeRateCountBackBlocks = 2
+
+	// defaultTestnetFeeRate is the default fee rate for testnet, 10 sat/byte
+	defaultTestnetFeeRate = 10
```

Comment:
I am wondering if this should be configurable by the zetacore through a policy , as in would there be any reason to update this value to unblock btc on testnet ever

Reply:
This one would be okay to be a non-state-based parameter. The edge case is when the actual network gas rate dropped to `0` (no traffic) in the past two blocks, `10` sat/vByte is to avoid posting a `0` rate that blocks outbound.